src/main/java/org/openbase/bco/bcozy/view/UnitMenu.java

```diff
@@ -40,11 +43,22 @@
 public class UnitMenu extends VBox {
 
     private final ContextSortingPane contextSortingPane;
+    private Pane powerTerminalSidebarPane;
     private final Label roomInfo;
+    private final Label energyInfo;
     private final ScrollPane verticalScrollPane;
     private TitledUnitPaneContainer titledPaneContainer;
     private FloatingButton fullscreenBtn;
     private final FloatingButton settingsBtn;
+    private final SVGGlyphIcon collapseIcon;
+    private FloatingButton collapseBtn;
+    private HBox collapseButtons;
+    private boolean maximized;
+    private HBox floatingButtons;
+    private MenuButton graphChoice;
+    private final HiddenSidesPane hiddenSidesPane;
+    private final double height;
```

Comment:
what are these properties for? in this the minimized or maximized size? Please encode in variable name. In general use fx properties for something like this.